src/twisted/web/_http2.py

```diff
@@ -1059,7 +1059,7 @@ def flowControlBlocked(self):
             self._producerProducing = False
 
     # Methods called by the consumer (usually an IRequest).
-    def writeHeaders(self, version, code, reason, headers):
+    def writeHeadersObject(self, version, code, reason, headers):
```

Comment:
The `H2Stream` class is private so non-backwards compatible changes seems OK.

Reply:
All good. Thanks.

My investigation on this api.

`H2Stream` is accesible via `H2Connection.streams{}`

`H2Connection` is accesible via
`twisted.web.http._GenericHTTPChannelProtocol._channel`

While `_GenericHTTPChannelProtocol` is made public via `twisted.web.http.HTTPFactory` the `_channel` member is still private and I could not find anywhere in the code where it might be exposed as public API.

So all good.